force_bdss/core_plugins/service_offers_plugin.py

```diff
@@ -0,0 +1,69 @@
+from envisage.api import ServiceOffer
+from traits.api import Instance, List, Interface
+
+from force_bdss.api import BaseExtensionPlugin
+
+
+class ServiceOffersPlugin(BaseExtensionPlugin):
```

Comment:
Considering name change to `ServiceOffersExtensionPlugin` in order to demonstrate inheritance from `BaseExtensionPlugin`